Dmitry_Predko/Hometask_6:nginx_in_practice/htpasswd

```diff
@@ -0,0 +1,3 @@
+#passwords for admin page
+
+admin:$apr1$o72j8rbF$sDrH5BfIBDMEzarDpDt8S0
```

Comment:
bad idea to put passwords in public access :) Even this one. 
the good solution is shell script which should be part of instruction to be run before building containers.